tests/test_scripts.py

```diff
@@ -297,7 +297,7 @@ def test_testpoi(tmpdir, script_runner):
     command = f'pyhf xml2json validation/xmlimport_input/config/example.xml --basedir validation/xmlimport_input/ --output-file {temp.strpath:s}'
     ret = script_runner.run(*shlex.split(command))
 
-    pois = [1.0, 0.5, 0.0]
+    pois = [1.0, 0.5, 0.001]
```

Comment:
While I'm not fundamentally against this very tiny change, is it understood why a POI of `0` results in a `RuntimeWarning: invalid value encountered in double_scalars`

```console
$ pytest tests/test_scripts.py -k test_testpoi
```
```pytb
...
  File "/home/feickert/Code/GitHub/pyhf/src/pyhf/infer/calculators.py", line 418, in _false_case
    teststat = (qmu - qmu_A) / (2 * self.sqrtqmuA_v)
RuntimeWarning: invalid value encountered in double_scalars

```


? I haven't stepped through enough things yet to understand why this is getting hit, but given this example's RuntimeWarning indicating that things are blowing up (given a super tiny denominator), does this particular example just happen to have an pathology that is causing this?

Reply:
Extremely small or extremely large values are occurring. In Linux, this crashes - on Mac, this passes and returns NaN for the obs and 1.0 for all expected CLs which means we're hitting infinities somewhere in the code.

See the failing test here: https://github.com/scikit-hep/pyhf/runs/8051304900?check_suite_focus=true . Not sure why this crashes the way it does, but as we don't care about what POI we're testing on this code, just that we test different ones and see different values, I changed it.

Reply:
The reason for this was Issue #1992, which was fixed in PR #1993.